scripts/utils/trading_strategy_helpers.js

```diff
@@ -394,44 +397,44 @@ const buildTransferApproveDepositTransaction = async function(
   web3,
   storeDepositsAsFile = false
 ) {
-  const fleetSize = slaves.length
+  const fleetSize = bracketAddresses.length
   assert(fleetSize % 2 == 0, "Fleet size must be a even number")
   const deposits = []
 
   let fundingTransaction = []
-  const FleetSizeDiv2 = fleetSize / 2
-  for (const i of Array(FleetSizeDiv2).keys()) {
+  const fleetSizeDiv2 = fleetSize / 2
```

Comment:
It would be awesome, if we could first generate the deposit file and then call the `buildTransferApproveDepositWithFunding` function with this deposit file. Less code duplication. But let's do it in a separate PR